app/src/main/java/lt/vilnius/tvarkau/API/ApiError.java

```diff
@@ -0,0 +1,5 @@
+package lt.vilnius.tvarkau.API;
```

Comment:
Convention for package naming is lowercase, single word. If need to use 2 words, then just `firstsecond`